L1Trigger/TrackFindingTracklet/src/TripletEngineUnit.cc

```diff
@@ -0,0 +1,113 @@
+#include "L1Trigger/TrackFindingTracklet/interface/TripletEngineUnit.h"
+#include "L1Trigger/TrackFindingTracklet/interface/VMStubsTEMemory.h"
```

Comment:
@cgsavard This could be removed, since it's already included in TripletEngineUnit.h.